lib/parsers/json.js

```diff
@@ -124,8 +124,14 @@ class JoiJsonSchemaParser {
 
   _getEnum(fieldDefn) {
     const enumList = fieldDefn[this.enumFieldName]
-    if (fieldDefn.flags && fieldDefn.flags.only && _.size(enumList) > 1) {
-      return _.uniq(enumList)
+    const filteredEnumList = enumList
+      ? enumList.filter(e => typeof e !== 'object'
```

Comment:
For this filter logic, I wonder if we should have a function named "isOverride" or "isNotOverride" and simply return !isOverride() or something similar which is more clearly tell the logic / idea behind?

Reply:
done